dev-libs/optix/optix-7.0.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Gentoo Authors
```

Comment:
Do we really need a `7.0.0` installation?
I would image a minor version bump should be backwards compatible, but then this *is* nvidia.

Reply:
media-gfx/blender-2.8.4 does not compile with dev-libs/optix-7.1.0, but does compile with dev-libs/optix-7.0.0. However, media-gfx/blender-2.9.0 compiles with dev-libs/optix-7.1.0. So, depending on which version of blender you want, you may need to stay < 7.1.0

Reply:
thanks nvidia...